CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Release Notes
 
+## [v2.1.2] (2024-11-04)
+
+* Check `.logfire` for creds to respect `'if-token-present'` setting by @sydney-runkle in [#561](https://github.com/pydantic/logfire/pull/561)
+
+## New Contributors
```

Comment:
i don't think we want this in the changelog, we haven't had it until now, but i can go in GH release notes if we want

Reply:
I can remove, but seems silly have to empty release notes?

Reply:
oh lol, you meant the contributors